projects/igniteui-angular/src/lib/date-common/calendar-container/calendar-container.component.html

```diff
@@ -3,10 +3,20 @@
         <button #closeButton *ngIf="cancelButtonLabel" igxButton="flat" igxRipple (click)="emitCalendarClose()">
             {{ cancelButtonLabel }}
         </button>
-        <button #todayButton *ngIf="todayButtonLabel" igxButton="flat" igxRipple (click)="selectToday()">
```

Comment:
not related to this PR but `emitCalendarClose` and `selectToday` are not the greatest name we could came up with. Could you change these?